packages/common/container-definitions/src/runtime.ts

```diff
@@ -119,10 +119,10 @@ export interface IRuntime extends IDisposable {
 	 *
 	 * @see {@link IContainer.getEntryPoint}
 	 *
-	 * @remarks The plan is that eventually IRuntime will no longer have a request() method, this method will no
-	 * longer be optional, and it will become the only way to access the entryPoint for the runtime.
+	 * @remarks The plan is that eventually IRuntime will no longer have a request() method and this method
```

Comment:
Same comment as in loader.ts. We can removed this remark since this function is now required.